ofl/yomogi/upstream.yaml

```diff
@@ -0,0 +1,5 @@
+branch: ver3.00
+files:
+  fonts/ttf/Yomogi-Regular.ttf: YomogiFont-Regular.ttf
```

Comment:
I think it needs to be updated here too for next updates

Reply:
I've updated this in the PR and upstream.  :)

(Changing font file names midstream is fun!)